x-pack/plugins/security_solution/cypress/integration/timeline_creation.spec.ts

```diff
@@ -46,11 +47,11 @@ import { openTimeline } from '../tasks/timelines';
 import { OVERVIEW_URL } from '../urls/navigation';
 
 // FLAKY: https://github.com/elastic/kibana/issues/79389
```

Comment:
Can you please remove this line and close the related ticket if is still opened?